.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
was there anything else we need to add here? Seems very empty compared to the react front ends so wasn't sure if the autogenerated gitignore was empty to begin with

Reply:
updating shortly

Reply:
ah cool, yeah the updated .gitignore looks more like what I expected